controllers/gateway/controller.go

```diff
@@ -30,6 +30,22 @@ type ControllerConfig struct {
 	ServiceName types.NamespacedName
 }
 
+// NewControllers sets up GatewayClass and Gateway controllers.
+func NewControllers(
+	ctx context.Context,
+	mgr ctrl.Manager,
+	pgr pomerium.GatewayReconciler,
+	config ControllerConfig,
+) error {
+	if err := NewGatewayClassController(mgr, config.ControllerName); err != nil {
+		return fmt.Errorf("couldn't create GatewayClass controller: %w", err)
```

Comment:
Error text here is inconsistent

Reply:
Thanks; updated the one below to match this one.